INSTALL

```diff
@@ -14,24 +14,13 @@ The source for the file utility + library is available from
     ftp://ftp.astron.com/pub/file/
 
 You will need a cryptographic library to support digests and signatures.
-This library may be libgcrypt, Mozilla NSS, OpenSSL or beecrypt.
+This library may be libgcrypt, Mozilla NSS or OpenSSL.
```

Comment:
This should be `libgcrypt or OpenSSL`, ne?

Reply:
Of course. Thanks for spotting.